tests/ui/test_display.py

```
@@ -0,0 +1,93 @@
+from rich.console import Console
+from rich.table import Table
+from rich.panel import Panel
+from datetime import datetime
+import feedparser
+from src.ui.display import display_feeds, display_latest_articles
+from src.models.feed import Feed
+
+class TestDisplay(unittest.TestCase):
+
+    @patch('src.ui.display.console')
+    def test_display_feeds(self, mock_console):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_display_feeds`

Add return type annotation: `None`

(ANN201)